### PR TITLE
Presets: show warning dialog when preset options list has not been op…

### PR DESCRIPTION
**Pull request by limonspb**
### Problem:
Very sad that the users still tend to miss the list of options.
For **some** presets it is **critical** to review the options and select.

This change checks if a preset has `#$ FORCE_OPTIONS_REVIEW: TRUE`
and if it has, then it forces the user to review options (to open drop down at least once).
image

## How to check this PR (before https://github.com/betaflight/firmware-presets/pull/173 is merged):
0. Download configurator from [here ](https://dev.azure.com/Betaflight/Betaflight%20Nightlies/_build/results?buildId=5745&view=artifacts&pathAsName=false&type=publishedArtifacts)if the current PR is not merged yet.
1. add preset source:
     https://github.com/limonspb/firmware-presets
     branch: mandate
2. find whoop preset by Justice
3. Try to pick it without opening options

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:6595]:
```suggestion
        "description": "Dialog text to prompt user to review options for the preset"
```

**Comment by SupaflyFPV** [comment]
looks like a good solution to teach people to take the extra step. It is also important for my presets they consider the options menu.

**Comment by limonspb** [comment]
> [[ Potato status confirmed - trying now with PR Configurator ]] So far, I have been able to apply that preset without any errors/flags/prompts, I'll try and review code to see what isn't behaving as expected there. This is the 10.8.0.db 83b2864a (2022-01-23) Configurator build I'm testing with.

plz get the Configurator from this PR, master do not have this change yet.

**Comment by TehllamaFPV** [comment]
Verified, correct behavior as intended.  Added to a local preset, behavior confirmed portable.
Looks good to me.

**Comment by limonspb** [comment]
changed the color for the options placeholder:
image

**Review by KarateBrot — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
